src/main/java/org/weakref/jmx/MBeanAttributeBuilder.java

```diff
@@ -18,20 +18,26 @@
 import static org.weakref.jmx.ReflectionUtils.isValidGetter;
 import static org.weakref.jmx.ReflectionUtils.isValidSetter;
 
+import com.google.common.base.Supplier;
+import com.google.common.base.Suppliers;
+import com.google.common.collect.ImmutableList;
+
 import javax.management.Descriptor;
 import javax.management.ImmutableDescriptor;
 import javax.management.MBeanAttributeInfo;
+
 import java.lang.reflect.Method;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.concurrent.TimeUnit;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class MBeanAttributeBuilder
```

Comment:
I hope that this class is not package-private only by mistake, and no-one uses it directly. Maybe we should make it so?